workflow/scripts/get_genome.py

```
@@ -0,0 +1,159 @@
+input_assembly = snakemake.params["assembly"]
+input_fasta = snakemake.params["fasta"]
+input_gff = snakemake.params["gff"]
+output_path = snakemake.output["path"]
+output_fasta = snakemake.output["fasta"]
+output_gff = snakemake.output["gff"]
+output_log = snakemake.log["path"]
+log = []
+error = []
+
+
+def check_fasta(input_fasta, log=[], error=[]):
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)